src/pugme.coffee

```diff
@@ -5,13 +5,14 @@
 #   None
 #
 # Configuration:
-#   None
+#   HUBOT_PUGBOMB_MAX - The most pugs you can bomb.  Default: -1 (unlimited)
 #
 # Commands:
 #   hubot pug me - Receive a pug
 #   hubot pug bomb N - get N pugs
 
 module.exports = (robot) ->
+  max = (process.env.HUBOT_PUGBOMB_MAX*1) or -1
```

Comment:
I found recently you can `parseInt` an undefined and get `NaN` back, and that's considered falsy. That means you can `parseInt(process.env.HUBOT_PUGBOMB_MAX) or -1`.